README.md

```diff
@@ -6,7 +6,7 @@
 
 Deis Workflow is the second major release (v2) of the Deis PaaS. If you are looking for the CoreOS-based PaaS visit [https://github.com/deis/deis](https://github.com/deis/deis).
 
-If you are looking to **get started** with **Deis Workflow** read the [Quick Start Guide](src/quickstart/index.md) in this repository or on [Read the Docs](https://docs-v2.readthedocs.org/en/latest/).
+If you are looking to **get started** with **Deis Workflow** please [Read the Docs](https://docs-v2.readthedocs.org/en/latest/).
```

Comment:
Actually, the brand new home of Workflow docs is https://deis.com/docs/workflow/, and ReadTheDocs.org is deprecated. @fagiani could you update this PR to something like:
```
To **get started** with **Deis Workflow** please read the [Quick Start Guide](https://deis.com/docs/workflow/quickstart/).
```
Which produces:

To **get started** with **Deis Workflow** please read the [Quick Start Guide](https://deis.com/docs/workflow/quickstart/).